content/authentication/troubleshooting-ssh/host-key-verification-failed.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Host key verification failed
+intro: 'As a security precaution, ssh keeps track of which hosts it has previously seen.'
+versions:
+  fpt: '*'
+  ghec: '*'
+topics:
+  - SSH
+shortTitle: Host key verification failed
+---
+
+This means that the key that the server to which your client is connecting presented a key that doesn't match the keys that your client has seen in the past.
+
+It could be because the server changed its keys unexpectedly -- this has happened -- in which case there should be a news reports from trustworthy sources (including the server's official blog) announcing the change.
```

Comment:
I think it would be useful to keep this article at least partly focused on GitHub, and explain how _GitHub_ handles changing its keys.

```suggestion
You may see this error if the server has changed its keys unexpectedly, in which case you should be able to find an official report from a trustworthy source announcing the change. If {% data variables.product.company_short %} changes its SSH host key, this will be announced on {% data variables.product.prodname_blog %} at [github.blog](https://github.blog/).
```